sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/http/HttpEndpointMethodDefinition.scala

```diff
@@ -80,6 +80,7 @@ object HttpEndpointMethodDefinition {
     for {
       methodDescriptor <- serviceDescriptor.getMethods.asScala.toSeq
       ruleBinding <- ruleBindings(serviceDescriptor, methodDescriptor)
+        .filter(_.pattern != HttpRule.Pattern.Empty)
```

Comment:
Hmm, this might be a problem. We might be using those 'internal' http rules to build the `DeferredCall` in the `KalixClient`. Ping @efgpinto as he probably knows it from the top of his head.

Reply:
The PR is passing, but could be that none of our tests are calling such a method. 
And that makes sense. A method without http rule is a method that is not expected to be exposed anyway.

Reply:
We are indeed using those HttpRule's to build the DeferredCall, but in those cases they will not be empty.. So, I can't see a problem with this. Also, we have a bunch of tests for using deferred calls both in IT and samples so the fact that CI passed is somewhat reassuring.